packages/app/src/app/pages/Dashboard/Components/NewTeamModal/index.tsx

```diff
@@ -51,7 +53,6 @@ const NewTeam: React.FC<NewTeamProps> = ({ step, hasNextStep, onClose }) => {
 
   return (
     <>
-      {' '}
       <Element padding={6}>
```

Comment:
Any chances we can set a height in this modal to prevent it from jumping around if there are no suggestions?